assets/html/scss/documenter/layout/_search.scss

```diff
@@ -17,8 +17,6 @@
       //   @include font-awesome;
       //   content: "";
       // }
```

Comment:
While we're at it, could we also remove this commented-out portion?

Reply:
Yeah, those two confused me too. There are some more above in `#documenter-search-results`.